physics/GFS_rrtmg_pre.F90

```diff
@@ -601,6 +604,15 @@ subroutine GFS_rrtmg_pre_run (im, levs, lm, lmk, lmp, n_var_lndp,        &
         enddo
        enddo
 
+      if(aero_dir_fdb) then ! add smoke/dust extinctions
+        do k = 1, LMK
+          do i = 1, IM
+            ! 550nm (~18000/cm)
```

Comment:
This is the extinction which will go into the aerosol feedback in physics? Only need 550 nm wavelength?

Reply:
Yes, we only calculate the 550nm wavelength additional effects by smoke and dust in SMOKE modules.